corehq/apps/hqcase/tests/test_case_update_api.py

```diff
@@ -330,10 +330,29 @@ def test_set_parent_by_bad_external_id(self):
         }).json()
         self.assertEqual(res['error'], "Could not find a case with external_id 'MISSING'")
 
-    def test_cannot_remove_child_case(self):
-        # Documenting that this does not yet work
+    def test_set_parent_missing_field(self):
+        parent_case = self._make_case()
+        res = self._create_case({
+            'case_type': 'match',
+            'case_name': 'Harmon/Luchenko',
+            'owner_id': 'harmon',
+            'indices': {
+                'parent': {
+                    'external_id': parent_case.external_id,
+                    'case_type': 'player',
+                    # 'relationship' is not specified
+                },
+            },
+        }).json()
+        self.assertEqual(res['error'], "Property relationship is required.")
```

Comment:
it would be useful to have more context on this error message - like where is it required:

`Property relationship is required when creating or updating case indices.`

Reply:
Good point - just pushed an update